test/recipes/95-test_external_tlsfuzzer.t

```diff
@@ -18,8 +18,8 @@ plan skip_all => "No external tests in this configuration"
     if disabled("external-tests");
 plan skip_all => "TLSFuzzer tests not available on Windows or VMS"
     if $^O =~ /^(VMS|MSWin32)$/;
-plan skip_all => "TLSFuzzer tests not supported in out of tree builds"
-    if bldtop_dir() ne srctop_dir();
```

Comment:
Aehm, sorry, is dropping the test for out of tree builds intentional?

Reply:
Yes, this requirement is not really relevant.

Reply:
Ah, okay then.